### PR TITLE
add request_time metric to access logs, BREAKING CHANGE

### PR DESCRIPTION
**Pull request by Keksoj**
As described in #1112, there was no `request_time` metric available (time between first byte received and last byte sent).
The value was computed, but wrongly assigned to the `response_time` metric.

This PR fixes it, and now, `response_time` has a better meaning and accurate value: the time between connecting to the backend and finishing the response.

We could even rename `request_time` to a more fitting `session_time`.

### CODE REVIEW

File: command/src/command.proto [hunk 1]
```diff
@@ -745,7 +745,7 @@ message ProtobufAccessLog {
     // round trip time for the backend (microseconds)
     optional uint64 server_rtt = 10;
     // time for the backend to respond (microseconds)
-    required uint64 response_time = 12;
+    optional uint64 response_time = 12;
```
Comment (FlorentinDUBOIS) [inline command/src/command.proto:748]: Should be move at the position 21 and removed from the 12. Keep it optional :) Like that we keep the ability to use an old client on newer version

File: command/src/command.proto [hunk 2]
```diff
@@ -760,6 +760,8 @@ message ProtobufAccessLog {
     required string tag = 18;
     // POSIX timestamp, nanoseconds
     required Uint128 time = 19;
+    // entire time between first byte received and last byte of the response
+    required uint64 request_time = 20;
```
Comment (FlorentinDUBOIS) [inline command/src/command.proto:764]: Should be optional as well, besides what happens if there is a trouble somewhere in the pipeline

**Review by FlorentinDUBOIS — CHANGES_REQUESTED** [review]

File: command/src/command.proto (hunk 2 above)
Comment (Keksoj) [reply command/src/command.proto:764]: An access log is produced at the very moment a pipeline (rather called "session") is over, be it successfully (`log_request_success`) or unsuccessfully (`log_request_error`).
The `request_time` metric is produced by a `SessionMetrics` object that computes time elapsed from the beginning of the session.
Do you think this behaviour is okay? Should we document it better?

We could add an additionaal line to this comment:

```
If a request ends abruptly before the last byte is transmitted,
the `request_time` produced is the time elapsed from the first byte received.
```

File: command/src/command.proto (hunk 1 above)
Comment (Keksoj) [reply command/src/command.proto]: done

File: command/src/command.proto (hunk 2 above)
Comment (Keksoj) [reply command/src/command.proto]: done!

**Review by llenotre — APPROVED** [review]
looks good to me

**Pull request merged**